cmd/prom-aggregation-gateway/main_test.go

```diff
@@ -154,17 +157,20 @@ func TestAggate(t *testing.T) {
 		{duplicateLabels, "", "", fmt.Errorf("%s", duplicateError), nil},
 		{reorderedLabels1, reorderedLabels2, reorderedLabelsResult, nil, nil},
 	} {
-		a := newAggate()
+		a := newAggate(3600000)
+		if c.b != "" {
+			c.a, c.b, c.want = fmt.Sprintf(c.a, now), fmt.Sprintf(c.b, now), fmt.Sprintf(c.want, now)
```

Comment:
Where is this checked or relied on?  I can't see that this test makes any use of the timestamp.